src/components/table/render/rowsActions/RowActions.tsx

```diff
@@ -1,68 +1,51 @@
 import React, { useState } from 'react';
-import { useConfig } from '@dhis2/app-runtime';
-import { MenuItem, IconList24 } from "@dhis2/ui";
-import { useGetEnrollmentForm, useParams } from '../../../../hooks';
-import { ModalComponent,  ModalContentUpdate } from '../../../modal';
-import { IconButton, Menu, Divider } from '@material-ui/core';
-import { RowActionsProps, RowActionsType } from '../../../../types/table/TableContentProps';
+import { IconEdit24 } from "@dhis2/ui";
+import style from './rowActions.module.css'
+import { IconButton,  Tooltip } from '@material-ui/core';
+import { useGetEnrollmentForm  } from '../../../../hooks';
+import { ModalComponent, ModalContentUpdate } from '../../../modal';
 import useGetSectionTypeLabel from '../../../../hooks/commons/useGetSectionTypeLabel';
+import { RowActionsProps, RowActionsType } from '../../../../types/table/TableContentProps';
 
 export default function RowActions(props: RowActionsProps) {
   const { row } = props;
-  const { baseUrl } = useConfig()
   const { sectionName } = useGetSectionTypeLabel();
-  const handleClose = () =>  setAnchorEl(null);
-  const [openModal, setOpenModal] = useState<boolean>(false);
   const { enrollmentsData } = useGetEnrollmentForm()
-  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
-  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => setAnchorEl(event.currentTarget);
-  const rowsActions = ({onOpenCapture, onEditStudent}: any) : RowActionsType[] => {
+  const [openModal, setOpenModal] = useState<boolean>(false);
+  
+  const rowsActions = ({onEditStudent} : any) : RowActionsType[] => {
       return [
-          { label: "1. Go to capture", divider: false, onClick: () => { onOpenCapture() }},
-          { label: "2. Edit Student", divider: false, onClick: () => { onEditStudent() }},
+          { label: "Edition", onClick: () => { onEditStudent() }, icon: <IconEdit24/>},
```

Comment:
Use a more descriptive label, such as "Edit student data"
@cnandza

Reply:
Can I combine the text with sectionName?